app/assets/js/scripts/lottery.js

```diff
@@ -176,11 +190,12 @@ class LotteryController {
     }
 
     _handleLotteryNotPicked() {
-        lotteryControllerLogger.info("I WAS NOT PICKED, pepehands");
+        lotteryControllerLogger.info("Someone was picked....and it wasn't you...pepehands");
         // alert("Sorry, you werent picked");
```

Comment:
moon2BOO

Reply:
Maybe need the special ascii emoji you prepared instead moon2S